tests/sparseml/pytorch/image_classification/resnet_test_recipe.md

```diff
@@ -0,0 +1,207 @@
+<!--
```

Comment:
this file is unnecessary IMO - copied from a zoo recipe contains bloat such as non-relevant modifiers and hyper parameters as well as the markdown.

We should be able to just move this into a constant string in python

```yaml
_TEST_RECIPE_STR = """
modifiers:
  - !ACDCPruningModifier
    compression_sparsity: 0.9
    start_epoch: 0.0
    end_epoch: 1.0
    global_sparsity: True 
    update_frequency: 1.0
    params: __ALL_PRUNABLE__

  - !QuantizationModifier
    start_epoch: 1.0
    disable_quantization_observer_epoch: 2.0
    freeze_bn_stats_epoch: 2.0
"""
```
should suffice

Reply:
Thanks for the minimal recipe, I would argue we should still keep this as a test file, point being a recipe file path is expected by the export script, we can always write the recipe to a temporary file on the fly during testing, let me know your thoughts on that